recipes/caffe2/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.8.dev" %}
+
+package:
+  name: caffe2
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/caffe2/caffe2.git
+  git_rev: master
+
+build:
+  number: 1
+  skip: True  # [win]
+
+requirements:
+  build:
+    - future
+    - glog
+    - numpy
+    - six
+    - eigen
+    - python <3
+    - protobuf ==3.4.0
+    - cmake
+    - leveldb
+    - lmdb
+    - snappy
+  run:
+    - future
+    - glog
+    - numpy
+    - six
+    - eigen
+    - python <3
+    - protobuf ==3.4.0
```

Comment:
Probably best to build with `3.4.0` but require `3.4.*`, since the ABI [is backwards compatible](https://abi-laboratory.pro/tracker/timeline/protobuf/) within minor versions and that should work.